addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/model/OshiSysteminfo.java

```diff
@@ -393,6 +427,20 @@ public DecimalType getSwapAvailablePercent() {
         }
     }
 
+    @Override
+    public DecimalType getSwapUsedPercent() {
+        long usedSwap = memory.getSwapUsed();
+        long totalSwap = memory.getSwapTotal();
+        BigDecimal usedPercent;
```

Comment:
Given the likeliness that totalSwap is actually <= 0 leave them used and total together (which also makes sense)
But please put the declaration of usedPercent with the assignment, in for example `getStorageUsedPercent` it already is like that, make it consistent for the others.